gdk/__init__.py

```diff
@@ -0,0 +1,6 @@
+import logging
+
+log_level = logging.INFO
+log_format = "[%(asctime)s] %(levelname)s - %(message)s"
+date_format = "%Y-%m-%d %H:%M:%S"
```

Comment:
Should we move these to constants file ?

Reply:
Done